### DISCUSSION TITLE
I cannot resolve the dependency for viewBinding

### DISCUSSION THREAD

**Discussion by IanisT227** [community/community#37797] [Other Feature Feedback, Questions, & Ideas]
Hi! I have implemented viewBinding in a previous project with your library and tried to do the same with a new project I began working on. However it Android Studio says Failed to resolve: com.github.Zhuinden:fragmentviewbindingdelegate-kt:1.0.2. I did not manage to find anything useful so I decided to post this

**Comment by jfullstackdev** [comment]
seems you are referring to this one 

https://github.com/Zhuinden/fragmentviewbindingdelegate-kt

you can create same questions at the issues tab there
